meeting-notes/2018/2018-12-10--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,218 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 December 10, 2018
+
+- **Lead:** @jacobheun
+- **Notetaker:** @achingbrain
+- **Attendees:**
+  - @vasco-santos
+  - @achingbrain
+  - @mcollina
+  - @daviddias
+  - @terichadbourne
+  - @pkafei
+  - @vmx
+- **Recording:** https://youtu.be/b5_g3rQJbAQ
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw (not attending)
+- Done:
+    - Created [benchmark tests matrix](https://gist.github.com/alanshaw/e8f49ff1875f0e256e7c45ddeef31498)
+    - PR for [stricter CID v1 to v0 conversion](https://github.com/multiformats/js-cid/pull/71)
+    - Fixed [streaming cat over HTTP API](https://github.com/ipfs/js-ipfs/pull/1760)
+    - PR for [CID version agnostic blockstore .get and .has](https://github.com/ipfs/js-ipfs-repo/pull/184)
+    - PR for [modularise files](https://github.com/ipfs/js-ipfs/pull/1772)
+    - PR for [addFromURL](https://github.com/ipfs/js-ipfs/pull/1773)
+- Blocked:
+    - Need reviews 🙏:
+        - [tests and fixes for CID version agnostic read and write](https://github.com/ipfs/js-ipfs/pull/1757)
+        - [streaming cat over http api](https://github.com/ipfs/js-ipfs/pull/1760)
+        - [modularise files](https://github.com/ipfs/js-ipfs/pull/1772)
+        - [add from url](https://github.com/ipfs/js-ipfs/pull/1773)
+- Next:
+    - Implement `addFromFs` and `addFromStream`
+    - Merge [DHT PR](https://github.com/ipfs/js-ipfs/pull/856) \o/
+    - Fix up and hopeful merge [`--cid-base` PR](https://github.com/ipfs/js-ipfs/pull/1552)
+    - Release an RC for [js-ipfs 0.34](https://github.com/ipfs/js-ipfs/issues/1721)
```

Comment:
It WILL be released before the holidays. Mostly you can help with reviews on the PRs above.

Things outstanding are:

* Merge DHT (@vasco-santos and I have this in hand)
* Implement `addFromFs` and `addFromStream` (Let me know if you'd like to volunteer)
* I need to fix up the `--cid-base` PR (I have this in hand)
* Noticed today... **WebUI needs updating as it doesn't work with the refactored files API** (Let me know if you'd like to volunteer, ping @olizilla?!)
* Finish writing the release notes for the blog post (I can do this but any help appreciated)

Then it is release time 🎈 I will shout, but I'd really appreciate help running the examples with the new release when the time comes.

cc @ipfs/javascript-team

Reply:
Sure: https://github.com/ipfs-shipyard/ipfs-webui/issues/901